ipv8/attestation/trustchain/community.py

```diff
@@ -334,9 +333,54 @@ def received_crawl_request(self, source_address, data):
             self.send_crawl_response(blocks[ind], payload.crawl_id, ind + 1, total_count, peer)
         self.logger.info("Sent %d blocks", total_count)
 
+    @synchronized
+    def recover_database(self):
```

Comment:
It is more like sanitization instead of recovering ;)

Reply:
I totally agree on removing the invalid blocks

Reply:
Good point, I'll rename that.